### DISCUSSION TITLE
What is required to show a empty graph (axes, labels, etcetera, just without data)?

### DISCUSSION THREAD

**Discussion by baum-sparton** [patrykandpatrick/vico#927] [Ideas]
### Question

I'm trying to display a graph which may sometimes not have data (not loaded yet, getting collected, etc.).

If I take the samples Chart1 and I comment out the lineSeries I get nothing but a white rectangle on the screen where the graph should be.  This looks awkward, and I'd like to still get the outline of the graph drawn (x and y axis labels, x and y axis, and some numbers and tick marks along the axes).  Thanks

### Vico version(s)

2.0.0-beta.1

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @baum-sparton. When there's no data, a placeholder is shown. It's blank by default, but you can customize it using `placeholder` parameter of `CartesianChartHost` function. You can't display a chart without adding any data, but you can work around it by adding a single entry with _y_ = 0 (and in the case of line charts with points, additionally hiding the points until the actual data is available).

I'm moving this to "Ideas", as a feature request.

**Comment by baum-sparton** [comment]
@Gowsky Thanks, both options worked.  The placeholder behaved as expected, and a series with a single (transparent) point resulted in a 'blank' graph.  I'm going with the single point series as that is the behavior desired by the end user in my situation.
